central/views/imagecve/view_test.go

```diff
@@ -78,6 +80,23 @@ func TestGetImageCVECore(t *testing.T) {
 		require.NoError(t, store.UpsertImage(ctx, image))
 	}
 
+	// Ensure that the image is stored and constructed as expected.
+	for idx, image := range images {
+		actual, found, err := store.GetImage(ctx, image.GetId())
+		require.NoError(t, err)
+		require.True(t, found)
+
+		cloned := actual.Clone()
+		// Adjust dynamic fields.
+		standardizeImages(image)
+		standardizeImages(cloned)
+		assert.EqualValues(t, image, cloned)
```

Comment:
Why do we need this check here? Except for the standardized fields, could the db store image differently than what was passed ?

Reply:
We want to make sure that the test is set up as expected. This is just an extra check.